RELEASE-NOTES.md

```diff
@@ -25,6 +25,7 @@
 * Support for Python remote execution via `snow stage execute` and `snow git execute` similar to existing EXECUTE IMMEDIATE support.
 * Added support for autocomplete in `--connection` flag.
 * Added `snow init` command, which supports initializing projects with external templates.
+* Added support for commands that yield multiple results
```

Comment:
Not sure if this is required, given it's an implementation detail and not a user feature.

Reply:
yeah i can take it out